bundles/org.openhab.binding.iotawatt/src/main/resources/OH-INF/i18n/iotawatt.properties

```diff
@@ -0,0 +1,3 @@
+# FIXME: please add all English translations to this file so the texts can be translated using Crowdin
```

Comment:
This placeholder needs to be updated by running the i18n plugin: `mvn i18n:generate-default-translations`